packages/ui5-proxy-middleware/.eslintignore

```diff
@@ -0,0 +1,3 @@
+test
+dist
+templates
```

Comment:
you don't need templates here, correct?

Reply:
No, I will remove it.

Reply:
Done

Reply:
I'm confused, we do need to ignore `templates/` during linting, don't we? :)

Reply:
But I don't have any templates in the project...

Reply:
Never mind then 😄